annotator-core/src/test/java/edu/ucr/cs/riple/core/CoreTest.java

```diff
@@ -426,6 +427,9 @@ public void staticBlockLocalVariableInitializationTest() {
             "   }",
             "}",
             "class B {",
+            "   {",
+            "      foo().hashCode();",
+            "   }",
```

Comment:
Do we want to test static initializer block support and instance initializer block in the same test? (if so, the test case might need renaming). Or is it perhaps better to have two separate test cases for ease of reading/debugging?

Reply:
I think the decision for now is to have them in one test until #137 is finalized, therefore I renamed the test name 344007b131ea98a58e54154d5db8e750f4c06983